test/e2e/cmd/run/run.go

```diff
@@ -261,6 +262,20 @@ func (h *helper) initTestSecrets() error {
 		return err
 	}
 
+	// Only initialize gcp credentials when running in CI
+	if os.Getenv("CI") != "" {
```

Comment:
In theory, only the CI system sets `CI=true`, nobody sets `CI=false`, however it would be more correct to check:
```suggestion
	if os.Getenv("CI") == "true" {
```